dotCMS/src/main/java/com/dotcms/rest/api/v1/site/CopyHostForm.java

```diff
@@ -0,0 +1,91 @@
+package com.dotcms.rest.api.v1.site;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CopyHostForm {
```

Comment:
doc

Reply:
We need to stop using the word "Host", and use "Site" instead

Reply:
Roger it, sure @jcastro-dotcms 
@freddyucv this is a draft (work on progress) I have half of the curl but not comited yet